flask/app/models.py

```diff
@@ -0,0 +1,48 @@
+from app import db
```

Comment:
Shouldn't we write the db in the models file? Also, do we need `db = SQLAlchemy(app)` or can we just use `db = SqlAlchemy()` ?

Reply:
We can move it to models. I think it should be db = SQLAlchemy(app)

Reply:
https://flask.palletsprojects.com/en/1.1.x/patterns/appfactories/#factories-extensions I'll try and implement something they have here as binding to the application does not look like it's recommended